pep-0646.rst

```diff
@@ -167,6 +170,9 @@ Type variable tuples are created with:
 
     Ts = TypeVarTuple('Ts')
 
+Using a Type Variable Tuple in Generic Classes
```

Comment:
I know we already talked about plural vs. singular, but in this case (and the next) I find the use of singular "Type Variable Tuple" with plural "Generic Classes" or "Functions" jarring. I propose to use plurals all the way.